AppCenter/AppCenterTests/MSCommonSchemaLogTests.m

```diff
@@ -212,6 +212,18 @@ - (void)testCSLogIsEqual {
   XCTAssertEqualObjects(anotherCommonSchemaLog, self.commonSchemaLog);
 }
 
+- (void)testOrderedDictionaryPerformance {
```

Comment:
Discussed offline with @ElektrojungeMS. 

In general we should exclude performance tests in CI, but it's probably fine for now since we have very few performance tests & they are quick & they never fail.